web/src/components/upload-indicator/upload-indicator.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import { IndeterminateBar, ProgressBar } from '@epam/loveship';
+import styles from './upload-indicator.module.scss';
+import { UploadProgressTracker } from 'connectors/documents-page-control-connector/use-fetch-with-tracking-of-upload-progress';
+
+export const UploadIndicator: FC<{
```

Comment:
It's possible to see flash "0%" when processing is finished - need to remove this behaviour.

Reply:
addressed by moving the reset logic to a setup function in the UploadIndicator component